lib/facebook_app_events.dart

```diff
@@ -144,11 +144,12 @@ class FacebookAppEvents {
   /// Parameter [registrationMethod] is used to specify the method the user has
   /// used to register for the app, e.g. "Facebook", "email", "Google", etc.
   /// See: https://developers.facebook.com/docs/reference/androidsdk/current/facebook/com/facebook/appevents/appeventsconstants.html/#eventnamecompletedregistration
-  Future<void> logCompletedRegistration({String? registrationMethod}) {
+  Future<void> logCompletedRegistration({String? registrationMethod, String? currency}) {
```

Comment:
If the parameter is mandatory it should be `required` and not nullable.

Reply:
made the changes